src/test/java/se/sundsvall/messaging/api/MessageResourceDigitalInvoiceTest.java

```diff
@@ -31,12 +35,11 @@
 class MessageResourceDigitalInvoiceTest {
 
 	private static final String MUNICIPALITY_ID = "2281";
-
 	private static final String URL = "/" + MUNICIPALITY_ID + "/digital-invoice";
-
 	private static final String ORIGIN_HEADER = "x-origin";
-
 	private static final String ORIGIN = "origin";
+	private static final String ORIGIN_ISSUER = "x-issuer";
```

Comment:
`ISSUER_HEADER`, to conform with `ORIGIN` headers?

Reply:
Yes of course, my bad. Changed the wrong part of name :) Will fix.